### ISSUE TITLE
BatchNormalization layer

### ISSUE THREAD

**Issue by jmduarte** [hls-fpga-machine-learning/hls4ml#71]
add support for `BatchNormalization` layer type:

https://github.com/keras-team/keras/blob/master/keras/layers/normalization.py#L16
